common/src/main/aidl/com/microsoft/identity/client/IMicrosoftAuthService.aidl

```diff
@@ -38,6 +38,8 @@ interface IMicrosoftAuthService {
 
     Intent getIntentForInteractiveRequest();
 
+    Intent getIntentForAccountTransferInteractiveRequest();
```

Comment:
add it at the bottom. 

there might be some weird side effects if the order of aidl on broker and MSAL/OneAuth side mismatches.

Reply:
Done